### DISCUSSION TITLE
1024 박한솔 블로그 생성

### DISCUSSION THREAD

**Discussion by hansol-woff** [Orm-camp/blog-feedback#27] [동글동글 3기 피드백]
[블로그 링크](https://velog.io/@incolore9/posts)  
  
저는 벨로그로 블로그를 생성했습니다. 
깔끔하고 글쓰기에만 집중할 수 있을 것 같아 해당 플랫폼으로 선택했습니다.  

완벽한 정보를 전달에는 작성에도 부담이 가서 우선은  경험담이나 소감 같은거 적으면서 재미있게 볼 수 있는 블로그가 됐으면 합니다..!

**Comment by ghost** [comment]
지윤환: 블로그 생성하시느라 고생 많으셨습니다. 
앞으로도 화이팅입니다!

**Comment by kimjigyun** [comment]
한솔님 멋있어요 따봉
